changelog/fragments/bump-1.26.yaml

```diff
@@ -0,0 +1,83 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      - Upgrade Kubernetes dependencies from `1.25` to `1.26`.
+      - Upgrade `sigs.k8s.io/controller-tools` from `v0.10.0` to `v0.11.3`. More info: https://github.com/kubernetes-sigs/controller-tools/releases
+      - Upgrade `sigs.k8s.io/controller-runtime` dependency from `v0.13.0` to `v0.14.5`. More info: https://github.com/kubernetes-sigs/controller-runtime/releases
+      - Upgrade `k8s.io/client-go` dependency from `v0.25.3` to `v0.26.2`.
+      - Upgrade `controller-gen` dependency from `v0.10.0` to `v0.11.1`.
+      - Upgrade `kube-rbac-proxy` from `v0.13.0` to `v0.13.1`
+      - Modify gov4alpha plugin to use `go/v4-alpha` from Kubebuilder
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Upgrade K8s versions to use 1.26
+      body: >
+        Regarding the upcoming major changes in the plugin, please note that `go/v3` is being deprecated by Kubebuilder hence Operator SDK would also be migrating to `go/v4` soon.
+        Following are the changes to be addressed in your project structure, `Makefile` and `go.mod` files:
+        1) [go/v3, ansible/v1, helm/v1] Update your `go.mod` file to upgrade the dependencies and run `go mod tidy` to download them
+        ```go
+        k8s.io/api v0.26.2
+        k8s.io/apiextensions-apiserver v0.26.2
+        k8s.io/apimachinery v0.26.2
+        k8s.io/cli-runtime v0.26.2
+        k8s.io/client-go v0.26.2
+        k8s.io/kubectl v0.26.2
+        sigs.k8s.io/controller-runtime v0.14.5
+        sigs.k8s.io/controller-tools v0.11.3
+        sigs.k8s.io/kubebuilder/v3 v3.9.1
+        ```
+        2) [go/v3, ansible/v1, helm/v1] With the bump to Kubebuilder 3.9.1, the `go/v3` projects are now scaffolded with 0.13.1 version of kube-rbac-proxy. Modify the version of kube-rbac-proxy in the following scaffolded files:
+        - config/default/manager_auth_proxy_patch.yaml
+        - bundle/manifests/memcached-operator.clusterserviceversion.yaml        
```

Comment:
This isn't rendering as expected in the docs https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.28.0/#upgrade-k8s-versions-to-use-126

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/7982347/230595118-860f713d-dca2-430b-96d6-3204a8fdff11.png">